### DISCUSSION TITLE
Tdarr breaks APT?

### DISCUSSION THREAD

**Discussion by dhenry437** [tteck/Proxmox#2310] [Help Needed] [answered]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

Hi

I have an issue where, right after spawning the Tdarr LXC (and before using it to perform any operations) I can happily use `apt`, mainly to install MKVToolNix.

After doing things in Tdarr i goto run any `apt` command and I get warnings saying `Operation not permitted` for `chown _apt:root /var/lib/apt/lists/partial` and `chmod 0700 /var/lib/apt/lists/partial` I also get the same two warnings for the `auxfiles` folder. If I continue anyway the whole thing errors out. (Sorry I don't have output to paste, I have since tried `rm`ing these folders and putting them back with the correct owner/permissions, this hasn't worked and has also changed the output slightly).

If I go look in the `/var/lib/apt/lists` directory the two folders are owned by `nobody` and in the group `root` the user nobody has an id of `65534`.

I have an id map set for the container in the file `/etc/pve/lxc/119.conf` on the proxmox host:
```
lxc.idmap: u 0 0 1
lxc.idmap: u 1 100000 65535
lxc.idmap: g 0 0 1
lxc.idmap: g 1 100000 65535
```
I have been using this on all my *arr apps so they can edit the media files that live on the host, so far this has worked fine but i wonder if there is a more proper way of doing this since the files are actually on a NAS, then the NAS is mounted to the host, and all the LXCs mount the hosts NAS mount.

Any help is appreciated

Thanks
Dan

### Which Linux distribution are you employing?

Debian 12

**Comment by tteck** [comment]
> - For suggestions, questions or feature/script requests, please share them in the [Discussions section.](https://github.com/tteck/Proxmox/discussions) 

Moving to Discussions...

**Comment by dhenry437** [comment]
revisiting this, the problem is definitely my `idmap`s in the lxc conf file, delete them and the problem disappears.

Weirdly the same `idmap`s work for Radarr and the rest, so I tried modifying the conf file to exclude all of the extra maps for USB devices and such, but still including my `idmap`s. No avail

**Comment by dhenry437** [comment, marked as answer]
just realised the container is privaleged, so no need for the `idmap`s. And by using the `idmap`s, `root` was no longer `root`
